src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.c

```diff
@@ -3,20 +3,33 @@
  * Overlay: ovl_Bg_Kin2_Bombwall
  * Description: Ocean Spider House - Bombable Wall
  */
-
 #include "z_bg_kin2_bombwall.h"
+#include "objects/object_kin2_obj/object_kin2_obj.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_10000000)
 
-#define THIS ((BgKin2Bombwall*)thisx)
+#define THIS ((BgKin2Bombwall*)(thisx))
 
 void BgKin2Bombwall_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
-const ActorInit Bg_Kin2_Bombwall_InitVars = {
+s32 BgKin2Bombwall_IsHitFromNearby(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SpawnEffects(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupWait(BgKin2Bombwall*);
+void BgKin2Bombwall_Wait(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupPlayCutscene(BgKin2Bombwall*);
+void BgKin2Bombwall_PlayCutscene(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupEndCutscene(BgKin2Bombwall*);
+void BgKin2Bombwall_EndCutscene(BgKin2Bombwall*, GlobalContext*);
```

Comment:
Name the arguments as in the function definitions. Also, we only include forward declarations that are necessary.

Reply:
Arguments named in definitions. Only _IsHitFromNearby and _SpawnEffects's forward declarations removed as all others are called earlier than their code in the file. Change will be visible in next push.